brambling/management/commands/send_daily_emails.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import traceback
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from brambling.mail import DailyDigestMailer
+from brambling.models import Person
+
+
+class Command(BaseCommand):
+    def get_recipients(self):
+        return Person.objects.filter(
+            notify_new_purchases=Person.NOTIFY_DAILY,
```

Comment:
True, but then we would have the problem of people getting emailed two hours later each day, which is weird. Down the road I could see that being a good method if we let people individually choose when their daily digest is sent. But for now I'd rather just stick to emailing everyone at the same time.